src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -4246,6 +4246,8 @@ def test_aks_create_with_windows(self, resource_group, resource_group_location):
                 "windows_admin_password": "replace-Password1234$",
                 "nodepool2_name": "npwin",
                 "ssh_key_value": self.generate_ssh_keys(),
+                "if_match": "*",
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=93139485&view=results) to validate the change
- test_aks_create_with_windows

Reply:
Thanks, latest successful live test run here https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=93593533&view=results